index.js

```diff
@@ -197,10 +197,11 @@ BabelClient.prototype.getAnnotations = function(token, querystringMap, callback)
  * @param {string} data.annotatedBy
  * @param {string} data.motiviatedBy
  * @param {string} data.annotatedAt
- *
- * @callback callback
+ * @param {object} options that control the request being made to babel.
+ * @param {boolean} options.xIngestSynchronously
+ * @param callback
  */
-BabelClient.prototype.createAnnotation = function(token, data, callback){
+BabelClient.prototype.createAnnotation = function(token, data, options, callback){
```

Comment:
i'll update the README and add the check to verify that if options in a function then its treated as the callback